### ISSUE TITLE
biz.k11i:xgboost-predictor:0.3.0 is now ported to the h2oai repo and released to Maven Central?

### ISSUE THREAD

**Issue by bbimber** [broadinstitute/gatk#7839]
Hello,

We have a java tool that requires GATK and our docker build began to fail a day or two ago. From what I can tell, it seems like biz.k11i:xgboost-predictor is being ported to ai.h2o:xgboost-predictor and perhaps biz.k11i:xgboost-predictor is falling off the mavenCentral() repos? This is a quick attempt to diagnose this. Are you seeing any problems in your builds? Do you have an alternate maven repo for biz.k11i:xgboost-predictor?

Thanks,
Ben

**Comment by lbergelson** [comment]
@bbimber I believe we resolve it from jcenter. There were some issues around it because we removed jcenter resolution from our build at one point when it was slated to shut down.  Testing didn't see any issue because it was being silently mirrored through our artifactory instance and we didn't realize that.  Jcenter changed their plans from shutting down to going into indefinite read only mode, so we re-added it.  

I think if you're seeing problems it's either because:
1. You are building on a version of gatk which removed jcenter
2. Your custom build doesn't resolve from jcenter
3. We have a new issue I don't know about yet.  

Can you rule out 1 and 2 before we start debugging 3?  

In your build.gradle you should should see ` jcenter()` in your `repositories {}` block.

It's very possible that we're relying on an outdated version and maybe we should update to a new one which isn't on the ill fated Jcenter only.  @TedBrookings any thoughts about that?  I think this is your dependency.

**Comment by TedBrookings** [comment]
It can be removed. I'm working in a new branch and full training was required, thus the predictor-only package is not beneficial.

**Comment by bbimber** [comment]
@lbergelson Thanks for the quick reply. I just re-added jcenter and it's working. I'll keep an eye on changes in GATK, since it would be slighter nicer to be on the newer version (or remove if @TedBrookings  is correct).
